manifests/mod/proxy.pp

```diff
@@ -27,7 +27,7 @@
   $apache_version = undef,
   $package_name   = undef,
   $proxy_via      = 'On',
-  $proxy_timeout  = '60',
+  $proxy_timeout  = $::apache::timeout,
```

Comment:
@gcoxmoz could you please try updating with following
$proxy_timeout  = ::apache::timeout
We tried and unit test cases ran clean. Now running the full acceptance suite to see if the test cases are running clean. Will let you know. Thank you for submitting the PR.

Reply:
<img width="869" alt="Screen Shot 2019-11-15 at 10 40 07" src="https://user-images.githubusercontent.com/20660680/69227284-7fb8fe80-0b79-11ea-9596-56c48b28818f.png">

Reply:
![Screen Shot 2019-11-20 at 09 38 46](https://user-images.githubusercontent.com/20660680/69227423-aecf7000-0b79-11ea-8bda-9fbc9ab0bba7.png)
The fix for redhat is available in master. If you could rebase your branch, we can get the PR run successfully on redhat8 too. Please let us know if you need more information.

Reply:
Messily updated per your suggestions (sorry, please squash if accepted).
I have no issue if you want to make edits in order to accept.  You have a much better handle on the actual code than I do.  Thanks!